app/models/user.rb

```diff
@@ -65,24 +64,16 @@ def self.tenant_id_clause(user_or_group)
     {table_name => {:id => users_ids}}
   end
 
-  def self.in_region
-    where(:region => my_region_number)
-  end
-
-  def self.in_my_region
-    where(:id => region_to_range(my_region_number))
-  end
-
   def self.find_by_userid(userid)
-    in_region.find_by(:userid => userid)
+    in_my_region.find_by(:userid => userid)
```

Comment:
This method makes me nervous. Very easy for someone to `User.find_by(:userid => userid)`
Guess only `default_scopes` would fix this. and I think we agree that is a no go.